packages/core/demo/data/area.ts

```diff
@@ -1,3 +1,6 @@
+import { timeFormat } from 'd3';
```

Comment:
We can't import stuff directly from `d3`, that'll ruin tree-shaking. Pls switch to the correct package, which I think is `d3-time-format`

Reply:
@theiliad have removed this, it wasn't being used and was left over when I was adding stuff in